src/mango_idx_view.erl

```diff
@@ -112,7 +112,25 @@ is_usable(Idx, Selector) ->
     % a member of the indexable fields of the selector.
     Columns = columns(Idx),
     Fields = indexable_fields(Selector),
-    lists:member(hd(Columns), Fields).
+    lists:member(hd(Columns), Fields) and not is_text_search(Selector).
```

Comment:
Why not to use short-circuit `andalso` operator here? If there is no such member in list, there seems no point to check if Selector is text search. Same case is about `Exists or is_text_search(C)` below.